src/site/content/en/blog/optimize-vitals-lighthouse/index.md

```diff
@@ -0,0 +1,365 @@
+---
+layout: post
+title: Optimizing Core Web Vitals using Lighthouse
```

Comment:
Do we we want to say "Web Vitals" here instead of "Core Web Vitals" given that Lighthouse exposes many of the non-core Web Vitals?

Reply:
Sure. I'll update this shortly to "Web Vitals"